core/src/test/scala/za/co/absa/fadb/statushandling/StatusHandlingSuite.scala

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2022 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.fadb.statushandling
+
+import org.scalatest.funsuite.AnyFunSuite
+import za.co.absa.fadb.DBFunctionFabric
+import za.co.absa.fadb.naming_conventions.{NamingConvention, SnakeCaseNaming}
+
+class StatusHandlingSuite extends AnyFunSuite {
+  test("Fields to select filled with default values") {
+    trait FooDBFunction extends DBFunctionFabric {
+      override def fieldsToSelect: Seq[String] = Seq("alpha", "beta")
```

Comment:
```suggestion
      override def fieldsToSelect: Seq[String] = Seq("Alpha", "beta")
```
Logic ignores first UpperCase letter.

Reply:
Actual the suggestion wouldn't work. On the inherited values the naming conversion doesn't run anymore.

On the other hand the `SnakeCaseNaming` is properly tested, I believe.

Reply:
That means that ignore of first capital is ok?

Reply:
It means, it should never come from the ancestor in this form. And if it does it's not the successor's responsibility to change/fix it. 
If it would be a part of 15 level class hierarchy , than the bottom level fields would be checked by the `NamingConvention` 15 times each. 😉